src/commands/link.ts

```diff
@@ -0,0 +1,78 @@
+import { CommandInteraction } from "discord.js";
+import { SlashCommandBuilder, hyperlink, hideLinkEmbed, userMention } from "@discordjs/builders";
+
+const LINKS: { [k: string]: { description: string; url: string } } = {
+  docs: {
+    description: "The Codewars Docs",
+    url: "https://docs.codewars.com/",
+  },
+  honor: {
+    description: "Honor | The Codewars Docs",
+    url: "https://docs.codewars.com/gamification/honor",
+  },
+  kata: {
+    description: "Kata | Codewars",
+    url: "https://www.codewars.com/kata",
+  },
+  katas: {
+    description: "Kata | Codewars",
+    url: "https://www.codewars.com/kata",
+  },
+  authoring: {
+    description: "Authoring Content | The Codewars Docs",
+    url: "https://docs.codewars.com/authoring",
+  },
+  rank: {
+    description: "Reviewing a Kata",
+    url: "https://docs.codewars.com/curation/kata",
+  },
+  ranking: {
+    description: "Reviewing a Kata",
+    url: "https://docs.codewars.com/curation/kata",
+  },
+  troubleshooting: {
+    description: "Troubleshooting Your Solution | The Codewars Docs",
+    url: "https://docs.codewars.com/training/troubleshooting",
+  },
+  debug: {
+    description: "Troubleshooting Your Solution | The Codewars Docs",
+    url: "https://docs.codewars.com/training/troubleshooting",
+  },
+  github: {
+    description: "Codewars on GitHub",
+    url: "https://github.com/codewars/",
+  },
+  clans: {
+    description: "Clans | The Codewars Docs",
+    url: "https://docs.codewars.com/community/following/#clans",
+  },
+  vim: {
+    description: "Editors | The Codewars Docs",
+    url: "https://docs.codewars.com/references/kata-trainer/#editors",
+  },
+};
+
+// link
+export const data = new SlashCommandBuilder()
+  .setName("link")
+  .setDescription("Link to a given topic")
+  .addStringOption((option) =>
+    option
+      .setName("topic")
+      .setDescription("The topic to link to")
+      .setRequired(true)
+      .addChoices(Object.keys(LINKS).map((k) => [k, k]))
```

Comment:
Was this (not using the description) intentional? Did it look weird?

Reply:
Apologies, it wasn't intentional, I didn't quite understand what you meant. You mean that we can specify a description for each choice on top of a key-value pair?

Reply:
I thought if we did
```typescript
  .addChoices(Object.keys(LINKS).map((k) => [LINKS[k].description, k]))
```
we can see the description when selecting choices.

Maybe I'm misremembering, but I thought it showed the "name" of the choice next to the "value".

Reply:
Ah, I see. Now that I tried it, it seems that the "key" doesn't even have to be unique :man_facepalming: 

I'll remove duplicate entries and open a PR shortly.